public/input.css

```diff
@@ -0,0 +1,203 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
```

Comment:
Everything looks good! This file is where you can make style classes that can be reused on JSX elements. 
A style class for a button would look like --> 
.btn{
    @apply w-full block text-sm rounded-full border-double border-2 border-purple-500 hover:bg-slate-300;
}

and then to apply it in the component you would use --> class="btn"